src/controller/java/src/chip/devicecontroller/ControllerParams.java

```diff
@@ -92,14 +101,15 @@ public static Builder newBuilder(OperationalKeyConfig operationalKeyConfig) {
   public static class Builder {
     private int udpListenPort = LEGACY_GLOBAL_CHIP_PORT + 1;
     private int controllerVendorId = 0xFFFF;
+    private int failsafeTimerSeconds = 30;
+    private boolean attemptNetworkScanWiFi = false;
```

Comment:
Why does this field default to false, but the thread field defaults to true? Additionally, there are no setters for these fields in the builder, nor documentation of their effects.

If these are solely for auto-commissioning, is it possible to isolate this into some wrapper class like `AutoCommissionerConfig`? It may help make the intent more clear as not all callers will be using auto-commissioner.

Reply:
setters are there but you might have been thrown off by the re-ordering of member variables in this PR. I added documentation for those parameters.

There is not really a way for a user of this Android SDK to not use the AutoCommissioner today. And since the CHIPDeviceController does have state handling for these scan network states, I think it makes sense to keep the parameters here for now.

Reply:
Please consider this for a follow-up as well

Reply:
Internally, we do not use the auto-commissioner. We use cluster commands directly.

It's a bit unclear to me what "attempt scan" actually means here. Are the scan results propagated somewhere? Not blocking but I don't understand the usage here.

Reply:
added all of these to https://github.com/project-chip/connectedhomeip/issues/21608